app/package.json

```diff
@@ -32,7 +32,7 @@
     "borc": "^2.1.0",
     "cardano-crypto.js": "5.2.0",
     "preact": "^8.2.7",
-    "trezor-connect": "7",
+    "trezor-connect": "^8.1.1",
```

Comment:
but you need also to update the package in yarn.lock to actually have the newer version used